src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UserStateAccessor.java

```diff
@@ -205,6 +205,10 @@ public final native JavaScriptObject getWindows() /*-{
          return this && this.windows || {"rBinDir":"","preferR64":true,"rExecutablePath":""};
       }-*/;
 
+      public final native JavaScriptObject getMacos() /*-{
```

Comment:
Is this used for the Copilot preference?

Reply:
This was generated when I ran the `generate-prefs` script -- perhaps it was missed with https://github.com/rstudio/rstudio/pull/13518?